src/main/java/frc/robot/subsystems/manipulator/ManipulatorIOSparkMax.java

```diff
@@ -0,0 +1,63 @@
+package frc.robot.subsystems.manipulator;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel;
+import com.revrobotics.RelativeEncoder;
+import com.revrobotics.SparkMaxPIDController;
+import edu.wpi.first.math.util.Units;
+import edu.wpi.first.wpilibj.RobotController;
+import frc.robot.subsystems.flywheel.FlywheelIO;
+
+public class ManipulatorIOSparkMax implements ManipulatorIO {
+    public static final double GEAR_RATIO = 1;
+    public static final int deviceId = 6; //TODO fill this in
+    private final CANSparkMax motor;
+    private final RelativeEncoder encoder;
+    private final SparkMaxPIDController pid;
+
+    public ManipulatorIOSparkMax(){
```

Comment:
if you want you can make motor id a parameter, but it will work this way as well